website/project/views/contributor.py

```diff
@@ -738,3 +740,12 @@ def claim_user_post(node, **kwargs):
         'email': email,
         'fullname': unclaimed_data['name']
     }
+
+@contributor_removed.connect
```

Comment:
This doesn't seem to be the proper place for this code. Since it's specific to OSF storage, it might make more sense to go in `website.addons.osfstorage.listeners` (new file). You will probably need to import that `listeners` module in `website/addons/osfstorage/__init__.py` to make sure that the signal gets connected when the app starts up.